audio/data.py

```diff
@@ -170,13 +181,15 @@ def open(self, item) -> AudioItem:
         mel = None
         if cfg.use_spectro:
             mel = MelSpectrogram(**asdict(cfg.sg_cfg))(signal.reshape(1, -1))
-            mel = mel.permute(0, 2, 1)
+            mel = mel.permute(0, 2, 1).squeeze()
             if cfg.to_db_scale:
                 mel = SpectrogramToDB(top_db=cfg.top_db)(mel)
             if cfg.cache:
                 os.makedirs(image_path.parent, exist_ok=True)
                 torch.save(mel, image_path)
-        return AudioItem(sig=signal.squeeze(), sr=samplerate, spectro=mel, path=item)
+            if cfg.standardize: mel = standardize(mel)
+            stacked = mel.expand(3,-1,-1)
+        return AudioItem(sig=signal.squeeze(), sr=samplerate, spectro=stacked, path=item)
```

Comment:
(Very Minor) Can stacked not just be assigned to mel? Also are we not giving the option to have 1 channel?